infrastructure/server/oidc_clients_forest_client.tf

```diff
@@ -45,12 +45,11 @@ resource "aws_cognito_user_pool_client" "test_forest_client_oidc_client" {
   callback_urls = [
     "https://oidcdebugggersecure-c6af30-dev.apps.gold.devops.gov.bc.ca/",
     "http://localhost:3000/dashboard",
-    "https://nr-forest-client-test-frontend.apps.silver.devops.gov.bc.ca/dashboard"
+    "https://forestclient-tst.nrs.gov.bc.ca/dashboard"
```

Comment:
update to use the vanity url as required, they don't need the openshift url anymore